src/pylhe/__init__.py

```diff
@@ -248,7 +248,7 @@ def _indent(elem, level=0):
 class LHEInit(dict):
     """Store the <init> block as dict."""
 
-    fieldnames = ["initInfo", "procInfo" "weightgroup", "LHEVersion"]
+    fieldnames = ["initInfo", "procInfoweightgroup", "LHEVersion"]
```

Comment:
```suggestion
    fieldnames = ["initInfo", "procInfo", "weightgroup", "LHEVersion"]
```

Woops. Looks like there's a bug here as of PR https://github.com/scikit-hep/pylhe/pull/233.

We should add a test to catch this and make a patch release.

cc @APN-Pucky @eduardo-rodrigues